notifications_team_report/notifications_team_report.py

```diff
@@ -39,28 +39,28 @@ def main():
     ap.add_argument('-r', '--resume-file', dest='resume_file', default=False,
         type=argparse.FileType('rb'), help="Read previous ILE data from this\
         file instead of retrieving it from the API.")
-    ap.add_argument('-v', dest='verbosity', action='count', default=0,
-        help="Logging verbosity (default: INFO-level messages).")
```

Comment:
This required the person running the script to enter `-v -v -v -v -v` in order to actually get debug logs...